.tool-versions

```diff
@@ -1,2 +1,2 @@
 poetry 1.1.13
-python 3.7.12
+python 3.9.12
```

Comment:
bumped py version to the latest supported by lambda. (this will also get rid of my pyarrow nightmares)